docs/reference/module-options.md

```diff
@@ -136,4 +138,257 @@ You might use that value as a fallback for user-editable fields.
 </h2>
 ```
 
-## Accessing module options
\ No newline at end of file
+## Options for all doc type modules
+
+Option settings in this section apply to all modules that extend `@apostrophecms/doc-type` ([doc type](glossary.md#doc) modules). These include all piece and page types.
+
+- [`adminOnly`](#adminonly)
+- [`autopublish`](#autopublish)
+- [`label`](#label)
+- [`localized`](#localized)
+- [`sort`](#sort)
+- [`slugPrefix`](#slugprefix)
+<!-- - [`contextBar`](#contextbar) -->
+
+### `adminOnly`
+
+<!-- TODO: link to permissions docs when available. -->
+If `true`, only users with admin-level permissions may edit this doc type. There is no default value.
+
+#### Example
+
+```javascript
+// modules/official-memo/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    adminOnly: true
+  },
+  // ...
+}
+```
+
+### `autopublish`
+
+Set `autopublish` to `true` to automatically publish any changes saved to docs of this type. There is then effectively no draft mode for this doc type. The core image and file modules use this option, for example, and it can be useful for such "utility" piece types that need to have a single, predictable state.
```

Comment:
This option eliminates the need for users to think about the distinction between draft and published content while preserving the possibility of translation for different locales. That is not quite a "single, predictable state." It is only a user interface convenience.